### PR TITLE
Feature collapsible panels

### PR DESCRIPTION
**Pull request by JPLachance**
* Add new collapsible panels from Materialize css
- Remove the old collapsable panels doc
* Rename components doc files properly
! Fix navigation generation
+ Add missing icons
- Remove unused md file
+ Add a splitted-layout util
+ Add mods for padding
+ Add a util file for borders
+ Add a modifier to remove thead border-top

### CODE REVIEW

File: lib/js/collapse.js [hunk 1]
```diff
@@ -1,211 +1,213 @@
...
-        var $target = getTargetFromTrigger($this)
-        var data = $target.data('bs.collapse')
-        var option = data ? 'toggle' : $this.data()
-
-        Plugin.call($target, option)
-    })
-
-}(jQuery);
\ No newline at end of file
+// DEPRECATED!!! Replaced by collapsible.js from Dogfalo/materialize
```
Comment (JPLachance) [inline lib/js/collapse.js]: Simply deprecated, ignore.

File: lib/js/transition.js [hunk 2]
```diff
@@ -1,63 +1,65 @@
...
-            delegateType: $.support.transition.end,
-            handle: function (e) {
-                if ($(e.target).is(this)) return e.handleObj.handler.apply(this, arguments)
-            }
-        }
-    })
-
-}(jQuery);
\ No newline at end of file
+// DEPRECATED!!! Replaced by collapsible.js from Dogfalo/materialize
```
Comment (JPLachance) [inline lib/js/transition.js]: Simply deprecated, ignore.

File: scss/components/modal.scss [hunk 3]
```diff
@@ -134,20 +134,10 @@ $base-modal-z-index: 1050;
 
       background-color: $pure-white;
 
-      &.mod-align-header {
```
Comment (JPLachance) [inline scss/components/modal.scss]: Simply moved and renamed.

File: scss/styleguide.md [hunk 4]
```diff
@@ -1,44 +0,0 @@
-[![Dependency Status](https://david-dm.org/Coveo/styleguide.svg?style=flat-square)](https://david-dm.org/Coveo/styleguide)
```
Comment (JPLachance) [inline scss/styleguide.md]: Unused...

File: docs/_includes/navigation.html [hunk 5]
```diff
@@ -1,9 +1,9 @@
 <nav class="navigation">
     <div class="navigation-menu">
         <ul class="navigation-menu-sections">
+            {% for pageInformation in site.pages %}
+            {% if pageInformation.title %}
```
Comment (GermainBergeron) [inline docs/_includes/navigation.html]: good catch

File: lib/js/transition.js (hunk 2 above)
Comment (GermainBergeron) [inline lib/js/transition.js]: What did you change in those files?

File: resources/icons/svg/arrow-bottom-rounded.svg [hunk 6]
```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12.6 7.2">
```
Comment (GermainBergeron) [inline resources/icons/svg/arrow-bottom-rounded.svg]: Why do you add the SVGs if you put them directly in the scss?

File: lib/js/transition.js (hunk 2 above)
Comment (JPLachance) [inline lib/js/transition.js]: I simply added the DEPRECATED comment.

File: resources/icons/svg/arrow-bottom-rounded.svg (hunk 6 above)
Comment (JPLachance) [inline resources/icons/svg/arrow-bottom-rounded.svg]: Because we will probably need them somewhere else in the future.

**Comment by wfortin** [comment]
🚢 it

File: scss/components/collapsible.scss [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+    line-height: $collapsible-height;
+    text-align: left;
+
+    background-color: $white;
+    background-image: url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAxMi42IDcuMiI+ICA8cGF0aCB0cmFuc2Zvcm09Im1hdHJpeCgtMSAwIDAgLTEgMTIuMzY2IDcuMDg2KSIgICAgICAgIGQ9Im0xMS40MjEgNy4wNGMtLjMgMC0uNS0uMS0uNy0uM2wtNC42LTQuNi00LjYgNC42Yy0uNC40LTEgLjQtMS40IDBzLS40LTEgMC0xLjRsNS4yLTUuMmMuNC0uNCAxLjItLjQgMS42IDBsNS4yIDUuMmMuNC40LjQgMSAwIDEuNC0uMi4yLS40LjMtLjcuMyIvPjwvc3ZnPg==');
+    background-repeat: no-repeat;
+    background-position: right $header-padding center;
+    background-size: 10px 10px;
+    border: none;
+    border-bottom: 1px solid $medium-grey;
```
Comment (pochretien) [inline scss/components/collapsible.scss]: i think we have a variable for the border size

File: scss/components/collapsible.scss [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+    font-size: 13px;
+    line-height: $collapsible-height;
+    text-align: left;
+
+    background-color: $white;
+    background-image: url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAxMi42IDcuMiI+ICA8cGF0aCB0cmFuc2Zvcm09Im1hdHJpeCgtMSAwIDAgLTEgMTIuMzY2IDcuMDg2KSIgICAgICAgIGQ9Im0xMS40MjEgNy4wNGMtLjMgMC0uNS0uMS0uNy0uM2wtNC42LTQuNi00LjYgNC42Yy0uNC40LTEgLjQtMS40IDBzLS40LTEgMC0xLjRsNS4yLTUuMmMuNC0uNCAxLjItLjQgMS42IDBsNS4yIDUuMmMuNC40LjQgMSAwIDEuNC0uMi4yLS40LjMtLjcuMyIvPjwvc3ZnPg==');
+    background-repeat: no-repeat;
+    background-position: right $header-padding center;
+    background-size: 10px 10px;
```
Comment (pochretien) [inline scss/components/collapsible.scss]: variable ?

File: scss/components/collapsible.scss [hunk 9]
```diff
@@ -0,0 +1,45 @@
+$collapsible-height: 50px;
+
+.collapsible {
+  .collapsible-header {
+    display: block;
+    width: 100%;
+    min-height: $collapsible-height;
+    padding: 0 $header-padding;
+
+    font-size: 13px;
```
Comment (pochretien) [inline scss/components/collapsible.scss]: not sure if we have a variable for the font-size too

File: scss/utility/layout.scss [hunk 10]
```diff
@@ -62,3 +62,19 @@
...
   visibility: hidden;
 }
+
+.splitted-layout {
+  flex-grow: 1;
+
+  display: flex;
+
+  .column {
+    flex: 1;
```
Comment (pochretien) [inline scss/utility/layout.scss]: i know it works too but for 1 variable, i think we can use flex-grow as above

File: scss/components/collapsible.scss (hunk 7 above)
Comment (JPLachance) [inline scss/components/collapsible.scss]: Done!

File: scss/components/collapsible.scss (hunk 8 above)
Comment (JPLachance) [inline scss/components/collapsible.scss]: Done!

File: scss/components/collapsible.scss (hunk 9 above)
Comment (JPLachance) [inline scss/components/collapsible.scss]: Done!

File: scss/utility/layout.scss (hunk 10 above)
Comment (JPLachance) [inline scss/utility/layout.scss]: I used `@extend .flex;` and `@extend .flex-auto;` instead.

**Pull request merged**